community/samples/README.md

```diff
@@ -1,5 +1,7 @@
 ## Sample apps
 
+Note: Community samples may or may not be maintained by their authors.
+
 | Sample Name | Knative Component | Description                                                | Language(s)                                                                                                                                                                                                                                                                                                                                                                                  |
 | ----------- | ----------------- | ---------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Hello World | Serving           | A quick introduction that highlights how to deploy an app. | [Clojure](./serving/helloworld-clojure/README.md), [Dart](./serving/helloworld-dart/README.md), [Elixir](./serving/helloworld-elixir/README.md), [Haskell](./serving/helloworld-haskell/README.md), [Rust](./serving/helloworld-rust/README.md), [Shell](./serving/helloworld-shell/README.md), [Swift](./serving/helloworld-swift/README.md), [Vertx](./serving/helloworld-vertx/README.md) |
```

Comment:
Can you please move .../helloworld-shell/... into the docs/serving/samples/README.md? (looks like this was missed when that sample got moved)